packages/@okta/vuepress-site/docs/reference/api/authorization-servers/index.md

```diff
@@ -661,11 +653,11 @@ When you use these API endpoints to create or modify a Scope resource, the respo
 ]
 ```
 
-#### Scope Properties
+#### Scope properties
 
 | Property                                 | Description                                                                                             | Type      | Default        | Required for create or update              |
 | :-------------------------------------   | :------------------------------------------------------------------------------------------------------ | :-------- | :------------- | :----------------------------              |
-| consent <ApiLifecycle access="ea" />     | Indicates whether a consent dialog is needed for the scope. Valid values: `REQUIRED`, `IMPLICIT`.       | Enum      | `IMPLICIT`     | True for update if this EA feature is enabled |
+| consent <ApiLifecycle access="ea" />     | Indicates whether a consent dialog is needed for the scope. Valid values: `REQUIRED`, `IMPLICIT`       | Enum      | `IMPLICIT`     | True for update if this EA feature is enabled |
```

Comment:
Since there's already a complete sentence, I'm not sure the second part can be a fragment. One possibility would be to change it to 'Valid values *are*:' and then end with a period.

Reply:
Actually that first sentence is a fragment. If it said it would be a complete sentence with an understood (You) as the subject: Indicate whether a consent dialog is needed for the scope.

But, we're doing fragments for table descriptions and endpoint descriptions.